buildSrc/src/main/kotlin/plugin/JetpackComposePlugin.kt

```diff
@@ -0,0 +1,50 @@
+package plugin
+
+import Versions
+import com.android.build.gradle.BaseExtension
+import org.gradle.api.JavaVersion
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.getByType
+import org.gradle.kotlin.dsl.withType
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+//See more in https://foso.github.io/Jetpack-Compose-Playground/general/getting_started/
+class JetpackComposePlugin : Plugin<Project> {
```

Comment:
this is neat!

Reply:
Yeah!!
I just try it out that it work or not when combine them into plugin.

Reply:
thank you!